README.md

```diff
@@ -54,3 +54,89 @@ This app is essentially a frontend microservice and should not be deployed by it
 To test ET3 in full, including RTF upload and form submission, use the [ET Full System](https://github.com/ministryofjustice/et-full-system).
 Before running the full system ensure ET3 is set to the branch or commit you wish to test.
 Please see the README for further details.
+
+## Environment Variables
+
+### DB_PORT
+
+Used to specify the database port.
+ 
+For example, used within the docker-compose configuration.
+
+### S3_PORT
+
+Used to specify the minio port.
+ 
+For example, used within the docker-compose configuration.
+
+### AZURE_STORAGE_BLOB_PORT
+
+Used to specify the azurite port.
+
+For example, used within the docker-compose configuration.
+
+### AWS_ACCESS_KEY_ID
+
+Used as an identifier to log into the minio portal.
+
+### AWS_SECRET_ACCESS_KEY
+
+Used as a password to log into the minio portal.
+
+### AWS_REGION
+
+Used to specify the AWS region for minio.
+
+For example, used to configure the bucket setup in minio.
+
+### S3_DIRECT_UPLOAD_BUCKET
+
+Used to specify the AWS bucket name for minio.
+
+For example, used to configure the bucket setup in minio.
+
+### AWS_ENDPOINT
+
+Used as the S3 endpoint, often taking in the `S3_PORT` as a parameter.
+
+For example, used to configure the bucket setup in minio.
+
+### AWS_S3_FORCE_PATH_STYLE
+
+Used as a boolean for the `force_path_style` option. 
+
+For example and further info, see [AWS Docs](https://docs.aws.amazon.com/sdkforruby/api/Aws/S3/Client.html).
+
+### SELENIUM_PORT
+
+Used to expose Selenium to a viewer.
+
+For example, used in conjunction with Zalenium to take a video of tests being run.
+
+### AZURE_STORAGE_BLOB_HOST
```

Comment:
Again, nothing to do with azure needs to be configurable - you are only using it within your test suite, not the app itself. Maybe an exception to that rule is the port - just in case the person running the test cannot use port 10000 ?